---
layout: default
name: Molecular barcode
theme: tuberculosis
pid: barcode
link: none
people:
    - fcoll
---

Strain-specific genomic diversity in the Mycobacterium tuberculosis complex (MTBC) is an important factor in pathogenesis that may affect virulence, transmissibility, host response and emergence of drug resistance. Several systems have been proposed to classify MTBC samples into distinct lineages and families, using molecular genotypes (e.g. spoligotyping or MIRU-VNTR), regions of difference (RD) and single nucleotide polymorphisms (SNPs). However, these systems lack accuracy or resolution due to the limited genetic variation studied or small number of samples used in their construction. We identified ~90k SNP across a global strain collection of 1,590 genomes. The SNP-based phylogeny was consistent with the gold-standard RD classification system and revealed fine-scale diversity. Of the ~7k strain-specific SNPs identified, a minimum set of 61 markers are proposed to discriminate accurately and robustly known circulating strains. This SNP barcode may be used to classify clinical isolates and evaluate tools to control TB, including therapeutics and vaccines whose effectiveness may vary by strain-type. 