---
layout: default
name: MTB Sequencing Variability
theme: tuberculosis
pid: mtb
link: https://github.com/jodyphelan/pathogenseq-scripts
people:
    - jphelan
---

To asses the analytical robustness of individual Mycobacterium tuberculosis DNA samples we sequenced replicates from the well characterised reference strain (H37Rv) and clinical isolates with resistance to up to 13 drugs. Sequencing was performed using the Illumina MiSeq and Ion Torrent PGMâ„¢ sequencing platforms. Two in-silico resistance calling pipelines were used to generate profiles. Results were compared to phenotypic drug susceptibility testing.
The MiSeq and Ion PGM system accurately predicted drug resistance profiles and there was high reproducibility between biological and technical sample replicates. MiSeq provided superior coverage in GC rich regions, which translated into incremental detection of putative genotypic drug-specific resistance, including for resistance to para-aminosalicylic acid and pyrazinamide. The TBProfiler bioinformatics pipeline was concordant with reported phenotypic susceptibility for all drugs tested except pyrazinamide and PAS, with an overall concordance of 95.3%.
In summary we have demonstrated high comparative reproducibility of two sequencing platforms. However, platform-specific variability in coverage of some genome regions may have implications for predicting resistance to specific drugs. These findings may have implications for future clinical practice and thus deserve further scrutiny, set within larger studies and using updated mutation libraries.