---
layout: default
name: P. vivax molecular barcode
theme: malaria
pid: vivax_barcode
link: none
people:
  - tgclark
  - scampino
  - edbenavente
---

Although Plasmodium vivax parasites are the predominant cause of malaria outside of sub-Saharan Africa, they not always prioritised by elimination programmes. P. vivax is resilient and poses challenges through its ability to re-emerge from dormancy in the human liver. With observed growing drug-resistance and the increasing reports of life-threatening infections, new tools to inform elimination efforts are needed. In order to halt transmission, we need to better understand the dynamics of transmission, the movement of parasites, and the reservoirs of infection in order to design targeted interventions. The use of molecular genetics and epidemiology for tracking and studying malaria parasite populations has been applied successfully in P. falciparum species and here we have developed a molecular genetic tool for P. vivax. By assembling a set of P. vivax whole genome sequences (n=433) spanning 17 countries, and applying a machine learning approach, we created a 71 SNP barcode with high predictive ability to identify geographic origin (>90%). Further, due to the inclusion of markers for within population variability, the barcode may also distinguish local transmission networks. By using P. vivax data from a low-transmission setting in Malaysia, we demonstrate the potential ability to infer outbreak events. By characterising the barcoding SNP genotypes in P. vivax DNA sourced from UK travellers to ten malaria endemic countries, we correctly predicted the geographic region of infection origin. We are now rolling out the 71 SNP barcode to inform targeted interventions towards elimination of this resilient human malaria. This work was funded by the UKRI MRC.