---
layout: post
pid: logo
hidden: true
author: tgclarke
link: https://pubmed.ncbi.nlm.nih.gov/36725857/
brief: The genetics underlying tuberculosis (TB) pathophysiology are poorly understood. Human genome-wide association studies have failed so far to reveal reproducible susceptibility loci, attributed in part to the influence of the underlying Mycobacterium tuberculosis (Mtb) bacterial genotype on the outcome of the infection. Several studies have found associations of human genetic polymorphisms with Mtb phylo-lineages, but studies analysing genome-genome interactions are needed. By implementing a phylogenetic tree-based Mtb-to-human analysis for 714 TB patients from Thailand, we identify eight putative genetic interaction points (P < 5 × 10-8) including human loci DAP and RIMS3, both linked to the IFNγ cytokine and host immune system, as well as FSTL5, previously associated with susceptibility to TB. Many of the corresponding Mtb markers are lineage specific. The genome-to-genome analysis reveals a complex interactome picture, supports host-pathogen adaptation and co-evolution in TB, and has potential applications to large-scale studies across many TB endemic populations matched for host-pathogen genomic diversity. 
---

The genetics underlying tuberculosis (TB) pathophysiology are poorly understood. Human genome-wide association studies have failed so far to reveal reproducible susceptibility loci, attributed in part to the influence of the underlying Mycobacterium tuberculosis (Mtb) bacterial genotype on the outcome of the infection. Several studies have found associations of human genetic polymorphisms with Mtb phylo-lineages, but studies analysing genome-genome interactions are needed. By implementing a phylogenetic tree-based Mtb-to-human analysis for 714 TB patients from Thailand, we identify eight putative genetic interaction points (P < 5 × 10-8) including human loci DAP and RIMS3, both linked to the IFNγ cytokine and host immune system, as well as FSTL5, previously associated with susceptibility to TB. Many of the corresponding Mtb markers are lineage specific. The genome-to-genome analysis reveals a complex interactome picture, supports host-pathogen adaptation and co-evolution in TB, and has potential applications to large-scale studies across many TB endemic populations matched for host-pathogen genomic diversity. 
