---
layout: default
name: Using deep learning to identify recent positive selection in malaria parasite sequence data
theme: 
  - malaria
  - software
pid: deepsweep
link: https://malariajournal.biomedcentral.com/articles/10.1186/s12936-021-03788-x
people:
  - emanko
  - jphelan
  - tgclark
  - scampino
  
highlighted: false
---

Using simulated genomic data, <i>DeepSweep</i> could detect recent sweeps with high predictive accuracy (areas under ROC curve > 0.95). <i>DeepSweep</i> was applied to <i>Plasmodium falciparum</i> (n = 1125; genome size 23 Mbp) and <i>Plasmodium vivax</i> (n = 368; genome size 29 Mbp) WGS data, and the genes identified overlapped with two established extended haplotype homozygosity methods (within-population iHS, across-population Rsb) (~ 60–75% overlap of hits at P < 0.0001). <i>DeepSweep</i> hits included regions proximal to known drug resistance loci for both <i>P. falciparum</i> (e.g. <i>pfcrt</i>, <i>pfdhps</i> and <i>pfmdr1</i>) and <i>P. vivax</i> (e.g. <i>pvmrp1</i>).