---
layout: default
name: Genomic analysis of hypervirulent <i>Klebsiella pneumoniae</i> reveals potential genetic markers
theme: 
  - amr
pid: klebsiellamarkers
link: https://www.nature.com/articles/s41598-022-17995-2
people:
  - aspadar
  - tgclark
  - scampino
  
highlighted: false
---

The majority of <i>Klebsiella pneumoniae</i> (Kp) infections are nosocomial, but a growing number of community-acquired infections are caused by hypervirulent strains (hvKp) characterised by liver invasion and rapid metastasis. Unlike nosocomial Kp infections, hvKp are generally susceptible to antibiotics. Due to the rapid progression of hvKp infections, timely and accurate diagnosis is required for effective treatment. To identify potential drivers of the hypervirulent phenotype, we performed a genome-wide association study (GWAS) analysis on single nucleotide variants and accessory genome loci across 79 publicly available Kp isolates collected from patients’ liver and a diverse global Kp dataset (n = 646). The GWAS analysis revealed 29 putative genes (P < 10–10) associated with higher risk of liver phenotype, including hypervirulence linked salmochelin iro (odds ratio (OR): 29.8) and aerobactin <i>iuc</i> (OR: 14.1) loci. A minority of liver isolates (n = 15, 19%) had neither of these siderophores nor any other shared biomarker, suggesting possible unknown drivers of hypervirulence and an intrinsic ability of Kp to invade the liver. Despite identifying potential novel loci linked to a liver invasive Kp phenotype, our work highlights the need for large-scale studies involving more sequence types to identify further hypervirulence biomarkers to assist clinical decision making.