---
layout: default
name: A phylogenomic approach for the analysis of colistin resistance-associated genes in <i>Klebsiella pneumoniae</i> 
theme: 
  - amr
pid: klebsiellaphylogenomic
link: https://www.sciencedirect.com/science/article/pii/S0924857922000814?via%3Dihub
people:
  - aspadar
  - jphelan
  - tgclark
  - scampino
  
highlighted: false
---

The emergence of carbapenemase-producing <i>Klebsiella pneumoniae</i> strains has triggered the use of old antibiotics such as colistin. This is driving the emergence of colistin resistance in multidrug-resistant strains that underlie life-threatening infections. This study analyses the mutational diversity of 22 genes associated with colistin resistance in 140 <i>K. pneumoniae</i> clinical isolates integrated in a high-resolution phylogenetic scenario. Colistin susceptibility was accessed by broth microdilution. A total of 98 isolates were susceptible and 16 were resistant, 10 of which were carbapenemase producers. Across the 22 genes examined, 171 non-synonymous mutations and 9 mutations associated with promoter regions were found. Eighty-five isolates had a truncation and/or deletion in at least one of the 22 genes. However, only seven mutations, the complete deletion of mgrB or insertion sequence (IS)-mediated disruption, were exclusively observed in resistant isolates. Four of these (<i>mgrB Ile13fs, pmrB Gly207Asp, phoQ His339Asp</i> and<i> ramA Ile28Met</i>) comprised novel mutations that are potentially involved in colistin resistance. One strain bore a <i>ISEcp1-blaCTX-M-15::mgrB</i> disruption, underlying co-resistance to third-generation cephalosporins and colistin. Moreover, the high-resolution phylogenetic context shows that most of the mutational diversity spans multiple phylogenetic clades, and most of the mutations previously associated with colistin resistance are clade-associated and present in susceptible isolates, showing no correlation with colistin resistance.