---
layout: default
name: TB-Profiler
pid: tbprofiler
theme: 
  - tuberculosis
  - software
link:  https://tbdr.lshtm.ac.uk/
people:
  - gnapier
  - jphelan
---

Drug resistance in Mycobacterium tuberculosis is primarily due to small polymorphisms within the genome. Current molecular tests examine limited number of loci. Whole genome sequencing has potential to overcome such problems but the complexity of data interpretation has, thus far, restricted its application. We have compiled a library of more than 1,300 mutations predictive of resistance for 15 anti-tuberculosis drugs (isoniazid, rifampicin, ethambutol, streptomycin, pyrazinamide, ethionamide, moxifloxacin, ofloxacin, amikacin, capreomycin, kanamycin, para-aminosalicylic acid, linezolid, clofazimine and bedaquiline).(Drug Resistance Mutations Library) To progress sequencing towards real time management of patients we have developed TB profiler that rapidly analyses raw sequence data and predicts resistance and lineage. In addition to identifying known drug resistance conferring mutations, the tool also identifies other mutations in the candidate regions. TB profiler processes fastq files at a rate of 80,000 sequence reads per second. Access to rapid, accurate and user friendly analytical tools like TB profiler are required to accelerate the uptake of next generation sequencing for detecting drug resistance, improving access to effective therapy under clinical settings. 
